docs/source_values.rst

```diff
@@ -178,12 +185,106 @@ In this example, the LED is lit only when the button is not pressed:
 
 Some tools combine the values of multiple sources:
 
-.. image:: images/combining_sources.*
+.. image:: images/source_values/combining_sources.*
     :align: center
 
 In this example, the LED is lit only if both buttons are pressed (like an
 `AND`_ gate):
```

Comment:
Might be worth repeating this example again but using `any_values`, just to illustrate the different kinds of thing that can be done?

Reply:
I'll add a note.